webpack/assets/javascripts/react_app/components/charts/StatisticsChartsListStyles.js

```diff
@@ -0,0 +1,27 @@
+export default {
```

Comment:
:-1: to this, for the following reasons:

- This is an unexpected place for styling, most developers wouldn't think of looking here.
- The issues described by the presentation are mostly not relevant to us or solved better with sass, and some stem from Facebook's extension of CSS.
- Sometimes global CSS is a good thing - for example, when you want all buttons to have the same style by default across the application.
- It feels like some of the issues mentioned in the presentation stem from a lack of understanding of CSS scoping and specificity.
- This may make sense for single page applications completely written in React, but for us this makes it practically impossible to override the styling from other places (for example, in a plugin or only for certain pages).
- Encapsulation and extraction can also be done by providing the styling in separate CSS files, like many libraries do, while allowing the consumer to easily change the styling using CSS if needed.

I believe this could be achieved better using regular SCSS and assigning classes to the various elements.